text/0001-transform-attribute-meta-parameter.md

```diff
@@ -1,8 +1,15 @@
 ---
-Start Date: 2014-08-14
-RFC PR: https://github.com/emberjs/rfcs/pull/1
-Ember Issue: https://github.com/emberjs/data/pull/4086
-
+start-date: 2014-08-14T00:00:00.000Z
```

Comment:
This metadata order is different than the template, do we care? I do find it less readable

Reply:
I don't see why metadata order matters 🤔 and what do you mean by "readable" in this context? Really we will be hoping to read most RFCs in the context of the app https://rfcs.emberjs.com/id/0001-transform-attribute-meta-parameter

right now that should be the default for all merged RFCs and in the future it will work for proposed ones too 👍

Reply:
Ok -- I'll probably always going to files, but so long as the order stays as it is in the template for new RFCs (and thus ones we will be editing), it seems fine.